app/src/ui/changes/changed-file.tsx

```diff
@@ -16,6 +16,7 @@ interface IChangedFileProps {
   readonly include: boolean | null
   readonly onIncludeChanged: (path: string, include: boolean) => void
   readonly onDiscardChanges: (path: string) => void
+  readonly onRevealInFileManager: (path: string) => void
```

Comment:
Can we document this please, specifically with information about how the path parameter is expected to be a path relative to the repository root.